source/EngineTests/CellConnectionTests.cpp

```diff
@@ -36,7 +36,8 @@ TEST_F(CellConnectionTests, decay)
 
 TEST_F(CellConnectionTests, addFirstConnection)
 {
-    auto data = CollectionDescription().addCells({
+    CollectionDescription data;
```

Comment:
Use single chained call:
`auto data = CollectionDescription().cells({CellDescription()... , CellDescription()..., ])`
Change the other places in the same way.